routes/reviewRouter.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+  saveReview,
+  editReview,
+  getReview,
+} = require("../controller/reviewController");
+const verifyPoodadakToken = require("../middlewares/verifyPoodadakToken");
+
+router.get("/:id", getReview);
```

Comment:
위에서 말씀드린 내용입니다.
```suggestion
router.get("/:reviewId", getReview);
```
이런식으로요!